hack/troubleshoot/README.md

```diff
@@ -46,7 +46,7 @@ In addition, macOS has the following requirements:
 Run the following command to run the script.
 
 ```bash
-bash -c "$(curl -fsSL https://raw.githubusercontent.com/dynatrace/dynatrace-operator/master/troubleshoot/troubleshoot.sh)"
+bash -c "bash -c $(curl -fsSL https://raw.githubusercontent.com/dynatrace/dynatrace-operator/master/hack/troubleshoot/troubleshoot.sh)"
```

Comment:
```suggestion
bash -c "$(curl -fsSL https://raw.githubusercontent.com/dynatrace/dynatrace-operator/master/hack/troubleshoot/troubleshoot.sh)"
```
It just complains because of that 2. `bash -c`